Gemfile

```diff
@@ -7,39 +7,19 @@ group :test do
   gem 'rspec-its', '~> 1.0'
   gem 'rspec-collection_matchers', '~> 1.0'
 
-  if RUBY_VERSION < '2.0'
-    # json 2.x requires ruby 2.0. Lock to 1.8
-    gem 'json', '= 1.8'
-    # json_pure 2.0.2 requires ruby 2.0, and 2.0.1 requires ruby 1.9. Lock to 1.8.3.
-    gem 'json_pure', '= 1.8.3'
-    # addressable 2.4.0 requires ruby 1.9.0. Lock to 2.3.8.
-    gem 'addressable', '= 2.3.8'
-    gem 'diff-lcs', '< 1.3'
-    gem 'rspec', '<= 3.4'
-  else
-    gem 'rspec', '~> 3.0'
-    gem 'json'
-  end
+  gem 'rspec', '~> 3.0'
+  gem 'json'
 
-  if RUBY_VERSION > '1.8'
-    # requires ruby 1.9+, on 1.8 we'll fall back to the old regex parsing
-    gem 'rspec-json_expectations', '~> 1.4'
-  end
+  gem 'rspec-json_expectations', '~> 1.4'
 
-  gem 'rubocop', '0.49.1' if RUBY_VERSION > '2.0'
   gem 'simplecov', :require => false if ENV['COVERAGE'] == 'yes'
 end
 
 group :development do
-  if RUBY_VERSION > '1.9'
-    # For Changelog generation
-    if RUBY_VERSION >= '2.2.2'
-      gem 'github_changelog_generator', :require => false
-    else
-      gem 'github_changelog_generator', '~> 1.13.0', :require => false
-      gem 'rack', '~> 1.0', :require => false
-    end
-
-    gem 'pry'
-  end
+    gem 'github_changelog_generator', require: false
+    gem 'faraday-retry', require: false
+    gem 'rubocop', '~> 1.6.1', require: false
+    gem 'rubocop-rspec', '~> 2.0.1', require: false
+    gem 'rubocop-performance', '~> 1.9.1', require: false
```

Comment:
In some places I've created a rubocop group.

Reply:
I like this - will add now.